### PR TITLE
Milestone 3: Portfolio Mobile  Version

### PR DESCRIPTION
**Pull request by ogagaoghene**
#Portfolio: Mobile Version 

#Description

1.  An update was made to the Mobile Version Skeleton
2. Work sections were added to the project
3. About myself section was added to the project
4. README.md file was updated with this new workflow

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,47 +4,185 @@
...
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">JavaScript</a></li>
+                </ul>
+                <ul class="btn">
+                    <li><a href="#">See Project</a></li>
+                </ul>
+            </div>
+            <div class="profile-card">
+                <div class="header-text">
+                    <img src="assets/me.png" alt=""/>
```
Comment (Lucas-Masaba) [inline index.html]: - [x] The About Myself text is supposed to be a heading with the `h1`, `h2, `h3`... tag but not an image. Please change this 👍

File: index.html [hunk 2]
```diff
@@ -4,47 +4,185 @@
...
+                    <img src="assets/icon-html.png" alt="" /> <br />
+                    <img src="assets/icon-css.png" alt="" />
+                </div>
+                <br/>
+                <div class="specials"><img src="assets/Frameworks.png" alt=""><a href="#"><i class='fas fa-greater-than icons'></i></a></div>
+                <div><img src="assets/separator.png" alt=""/></div>
+                <div class="specials"><img src="assets/Skills.png" alt=""/><a href="#"><i class='fas fa-greater-than icons'></i></a></div>
+                <div><img src="assets/separator.png"  alt=""/></div>
+            </div>
+        </section>
```
Comment (Lucas-Masaba) [inline index.html]: - [x] The About Myself should be wrapped in a `section` tag.

File: index.html [hunk 3]
```diff
@@ -4,47 +4,185 @@
...
+                <ul class="btn">
+                    <li><a href="#">Get Resume</a></li>
+                </ul>
+                <p>&nbsp;</p>
+                <div class="specials"><img src="assets/Languages.png" alt=""/><a href="#"><i class="fa fa-angle-down icons" aria-hidden="true"></i></a></div>
+                <br />
+                <div>
+                    <img src="assets/icon-js.png" alt=""/> <br />
+                    <img src="assets/icon-html.png" alt="" /> <br />
+                    <img src="assets/icon-css.png" alt="" />
```
Comment (Lucas-Masaba) [inline index.html]: - [x] This is a list of languages and you are required to enclose this in a `ul` tag with each language wrapped in a `li` tag

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @ogagaoghene  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 
- Good looking README.md 🏄 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -4,47 +4,192 @@
...
+                  </ul>
+                </div>
+                <br/>
+                <div class="specials"><img src="assets/Frameworks.png" alt=""><a href="#"><i class='fas fa-greater-than icons'></i></a></div>
+                <div><img src="assets/separator.png" alt=""/></div>
+                <div class="specials"><img src="assets/Skills.png" alt=""/><a href="#"><i class='fas fa-greater-than icons'></i></a></div>
+                <div><img src="assets/separator.png"  alt=""/></div>
+            </div>
+          </section>
+        </section>
```
Comment (muneebulrehman) [inline index.html]: Both work and the about-me section are supposed to be two different sections but you have wrapped both in a single section. Please wrap both of them in two different `section` tags.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @ogagaoghene , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

Consider updating your README file for the below sections.
github


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ogagaoghene** [comment]
I have done as advised. I hope for a positive review to enable me to advance to the next stage. Thanks in anticipation and for your professional input. I am eternally grateful to you.

File: index.html (hunk 4 above)
Comment (ogagaoghene) [reply index.html]: I have separated the work section from the about-me section as recommended. Thank you so very much for your professional review. I look forward to receiving positive feedback from you so I can prepare for the next step. Thanks in anticipation and much appreciation from me to you

**Review by ishimwezachee — APPROVED** [review]
Hi @ogagaoghene, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ogagaoghene** [comment]
Thanks for your professional code reviews.

**Pull request merged**